packages/css/src/components/link-list/link-list.scss

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+@mixin reset-list {
+  box-sizing: border-box;
+  list-style: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+  -webkit-text-size-adjust: 100%;
+}
+
+.amsterdam-link-list {
+  @include reset-list;
+
+  display: grid;
+  gap: var(--amsterdam-link-list-gap);
+}
+
+.amsterdam-link-list__link {
+  align-items: flex-start;
+  color: var(--amsterdam-link-list-link-color);
+  display: inline-flex;
+  font-family: var(--amsterdam-link-list-link-font-family);
+  font-size: var(--amsterdam-link-list-link-spacious-medium-font-size);
+  gap: var(--amsterdam-link-list-link-gap);
+  line-height: var(--amsterdam-link-list-link-spacious-medium-line-height);
+  text-decoration-line: var(--amsterdam-link-list-link-text-decoration-line);
+  text-decoration-thickness: var(--amsterdam-link-list-link-text-decoration-thickness);
+  text-underline-offset: var(--amsterdam-link-list-link-text-underline-offset);
+  vertical-align: middle;
+
+  .amsterdam-theme--compact & {
+    font-size: var(--amsterdam-link-list-link-compact-medium-font-size);
+    line-height: var(--amsterdam-link-list-link-compact-medium-line-height);
+  }
+
+  &:hover {
+    text-decoration-line: var(--amsterdam-link-list-link-hover-text-decoration-line);
+  }
+
+  &:visited {
+    color: var(--amsterdam-link-list-link-visited-color);
+  }
```

Comment:
I don't think this is necessary. This was only necessary because we set the visited color in the Link component, but we don't do that here.